dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
interpreting well defined rdf blank nodes 
as neo4j anonymous nodes seems the only sensible thing to do.  
If we are relying on the first character of a post curi-prefix/iri-fragment identifier 
to be an underscore to determine if a scigraph node is anonymous, then that needs addressing.    
If neo4j/scigraph are already correctly interpreting standard blank nodes then nothing would break until we actually emitted skolems. 
Anonymous nodes should most likely already be getting generated for any iri   
with the pattern "_host_ /.well-known/genid/"  as they could already be occurring in the wild.